docs/tutorials/05_Sampling_potential_energy_surfaces.ipynb

```diff
@@ -0,0 +1,535 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sampling the potential energy surface "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction \n",
+    "\n",
+    "This interactive notebook demonstrates how to utilize the Potential Energy Surface (PES) samplers algorithm of qiskit chemistry to generate the dissociation profile of a molecule. We use the Born-Oppenhemier Potential Energy Surface (BOPES)and demonstrate how to exploit bootstrapping and extrapolation to reduce the total number of function evaluations in computing the PES using the Variational Quantum Eigensolver (VQE). \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import common packages\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "from functools import partial\n",
+    "\n",
+    "# qiskit\n",
+    "from qiskit.utils import QuantumInstance\n",
+    "from qiskit import Aer\n",
+    "from qiskit.algorithms.minimum_eigen_solvers import NumPyMinimumEigensolver, VQE\n",
+    "from qiskit.algorithms.optimizers import SLSQP\n",
+    "from qiskit.circuit.library import ExcitationPreserving\n",
+    "from qiskit import BasicAer\n",
+    "from qiskit.algorithms import NumPyMinimumEigensolver, VQE\n",
+    "from qiskit.algorithms.optimizers import SLSQP\n",
+    "\n",
+    "# qiskit nature imports\n",
+    "from qiskit_nature.problems.second_quantization import ElectronicStructureProblem\n",
+    "from qiskit_nature.operators.second_quantization.qubit_converter import QubitConverter\n",
+    "from qiskit_nature.mappers.second_quantization import JordanWignerMapper\n",
+    "from qiskit_nature.algorithms.ground_state_solvers import GroundStateEigensolver\n",
+    "from qiskit_nature.drivers import PySCFDriver, UnitsType, Molecule\n",
+    "from qiskit_nature.algorithms.pes_samplers import BOPESSampler, Extrapolator\n",
+    "\n",
+    "import warnings\n",
+    "warnings.simplefilter('ignore', np.RankWarning)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we use the H2 molecule as a model system for testing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "driver = PySCFDriver()\n",
```

Comment:
What does this cell do as such. It creates a default driver, which while it builds out an H2 by default is less obvious. And nothing is solved here with it and then in next cell driver is setup with an H2 this overriding this one. Maybe the groundstate solver creation should be delayed until its needed by the BOPESampler which may make understanding what code one needs etc a little easier

Reply:
updated it.